terraform/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "api-domain-name" {
+  value       = module.api.api-domain-name
+  description = "Custom domain name for the API"
+  sensitive   = true
+}
+
+output "api-gateway-url" {
+  value       = module.api.api-gateway-url
+  description = "Cloud run gateway URL for the API"
+  sensitive   = true
+}
```

Comment:
Do these need to be sensitive? I think it would be useful to see them. It would need updating in the Terraform module too.

Reply:
I don't think they need to be, was just being a bit conservative :-) I will update them to not be sensitive.